core/trino-main/src/test/java/io/trino/server/security/oauth2/TestOAuth2Config.java

```diff
@@ -14,20 +14,33 @@
 package io.trino.server.security.oauth2;
 
 import com.google.common.collect.ImmutableMap;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigSecuritySensitive;
 import io.airlift.units.Duration;
 import org.testng.annotations.Test;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Map;
 
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
 import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
 import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
 
 public class TestOAuth2Config
 {
+    private static List<String> findSensitiveConfigName(Class configClass)
```

Comment:
Please move this method to the bottom, below the usage.

Reply:
I don't think there should be a test for this. Its just testing annotation existence.

Reply:
I was thinking the same way. But one may say that entire `TestOAuth2Config` goes into the same bucket. I am fine either way.

Reply:
It's an existing convention. We can consider changing the convention either way -- like removing those `Test*Config` classes, or adding assertions for `@ConfigSecuritySensitive` existence. 
If we want to do so, we should have some sort of high-level agreement.

Reply:
@shawnzhu Is this ok to you if just drop this test?

Reply:
@kokosing sure! It's just a test to make sure `ConfigSecuritySensitive` annotation is added to the desired method, as a requirement.

I'd hope to move the method `findSensitiveConfigName()` to somewhere like `trino-testing` or even upperstream airlift (don't know where it is)